edge/pkg/common/util/config.go

```diff
@@ -41,7 +41,7 @@ func GenerateTestCertificate(path string, certFileName string, keyFileName strin
 		SerialNumber:          big.NewInt(1234),
 		Subject: pkix.Name{
 			Country:      []string{"test"},
-			Organization: []string{"testor"},
+			Organization: []string{"tester"},
```

Comment:
It seems that the two words mean the same thing.

Reply:
![1637909104(1)](https://user-images.githubusercontent.com/86346333/143538570-58f1766e-fa92-4371-839a-cb6cfd2beea1.png)
It seems that there isn't such a word 0.0

Reply:
;-)